form.js

```diff
@@ -0,0 +1,21 @@
+const containerform = document.querySelector('.input');
```

Comment:
- [ ] If a user submits the contact form with the email field containing only lowercase characters the form is sent normally. for now, the form doesn't submit at all, kindly fix this so the form is sent when the user clicks the submit button